proxies/live/apiproxy/policies/AssignMessage.OAuthPolicyOperationOutcomeErrorResponse.xml

```diff
@@ -0,0 +1,8 @@
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.OAuthPolicyOperationOutcomeErrorResponse">
+    <Set>
+        <StatusCode>401</StatusCode>
+        <Payload contentType="application/fhir+json" variablePrefix="%" variableSuffix="#">{ "meta": { "lastUpdated": "{current_timestamp}", "profile": [ "https://www.hl7.org/fhir/R4/operationoutcome" ] }, "resourceType": "OperationOutcome", "issue": [ { "severity": "error", "code": "forbidden", "details": { "coding": [ { "system": "https://fhir.nhs.uk/CodeSystem/eRS-APIErrorCode", "code": "ACCESS_DENIED" } ] }, "diagnostics": "%faultstring#" } ] }</Payload>
```

Comment:
the profile should be https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome

Reply:
the code could be "login" instead of forbidden, but need to raise a question against the spec first.

Reply:
https://fhir.nhs.uk/CodeSystem/NHSD-API-ErrorOrWarningCode instead of https://fhir.nhs.uk/CodeSystem/eRS-APIErrorCode